Dockerfile

```diff
@@ -12,4 +12,9 @@ WORKDIR /app
 RUN npm install -g yarn@1.5.1
 RUN yarn install
 
-CMD yarn start
+# TODO: Run below on deploys
+# RUN yarn setup_k8s
```

Comment:
Basically, the command on prod/staging should be `yarn setup_k8s && IS_K8S=true yarn start`, vs just regular ol' `yarn start` on Circle.

Reply:
My question is more: how do we get a different command to run here on Circle, vs on k8s? Also locally, etc. I'm just not sure the Docker semantics to do that (if it makes sense to).

Reply:
After sleeping on it, I guess this breaks containerization. We shouldn't have different commands on deploy boxes vs local vs Circle CI.

So I guess the real question, is: how come the below works fine locally? Yet on Circle was crashing: https://circleci.com/gh/artsy/force/5898

The error was:
```
scripts/setup_k8s.sh: 9: scripts/setup_k8s.sh: Syntax error: "(" unexpected (expecting "then")
error An unexpected error occurred: "Command failed.
Exit code: 2
```

Some sort of bash version mismatch that doesn't like the script? Script works fine on my machine.

Reply:
You can of course have the same command operate slightly differently based on ENV (with the container's env provided in .yml or ConfigMap).

Not sure about the syntax error though... I can't imagine why building would succeed locally but fail on Circle. Maybe there are local volume mappings interacting with this?

Reply:
> You can of course have the same command operate slightly differently based on ENV (with the container's env provided in .yml or ConfigMap).

I'll look into that to try to get past the Circle issue. I was thinking a bash version mismatch of some kind.

Reply:
Are you saying the command works when you run it directly in a terminal on your laptop, or that it works when you build the Dockerfile (like `hokusai build`)? The latter runs on the exact base image as presumably happens in Circle, so a version mismatch is unexpected.

Reply:
It works when run directly in a terminal, ie- `yarn setup_k8s` correctly builds + fingerprints the assets.

Reply:
Ah... I would debug inside the container. That should perfectly replicate the Circle env.

Reply:
`package.json` commands are just aliases and it helps other devs see whats possible without having to dig into code or remember to include environment variables. 

```js
"scripts": {
  'yarn start': 'node index.js',
  'yarn start:prod': 'NODE_ENV=production node index.js --plus --some --other --stuff'
  'yarn start:k8': 'hokousai up'`
}
```